Point_set_processing_3/include/CGAL/Point_with_normal_3.h

```diff
@@ -147,7 +147,7 @@ struct Normal_of_point_with_normal_pmap
   typedef Normal_of_point_with_normal_pmap<Gt> Self;
   /// \name Put/get free functions
   /// @{
-  friend const reference get(const Self&,const key_type& k) {return k.normal();}
+  friend const value_type& get(const Self&,const key_type& k) {return k.normal();}
   friend reference get(const Self&, key_type& k) {return k.normal();}
   friend void put(const Self&,key_type& k, const value_type& v) {k.normal()=v;}
```

Comment:
`reference` should be `typedef value_type& reference;`, and both overloads should return the `reference` type.